### PR TITLE
Added common test to rqt_py_common

### PR DESCRIPTION
**Pull request by ahcorde**

### CODE REVIEW

**Comment by ahcorde** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/21381/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/15687/)
* Linux-rhel [Build Status](http://ci.ros2.org/job/ci_linux-rhel/890/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/22117/)

File: rqt_py_common/src/rqt_py_common/item_delegates.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ class DelegateUtil(object):
     def _get_hierarchy_level(model_index):
         hierarchyLevel = 1
         seek_root = model_index
-        while(seek_root.parent() != QModelIndex()):
+        while (seek_root.parent() != QModelIndex()):
```
Comment (clalancette) [inline rqt_py_common/src/rqt_py_common/item_delegates.py]: I'll point out that the parentheses aren't required at all here:

```suggestion
        while seek_root.parent() != QModelIndex():
```

File: rqt_py_common/src/rqt_py_common/message_helpers.py [hunk 2]
```diff
@@ -139,8 +139,8 @@ def get_all_rosidl_types_of_type(rosidl_type):
     :returns: a dictionary mapping packages to the rosidl interfaces of a specific type
     """
     if rosidl_type not in ROSIDL_FILTERS:
-        raise ValueError('Invalid rosidl_type type "{}". Needs to be one of {}'.format(
-            filter_type, ROSIDL_FILTERS.keys()))
+        raise ValueError('Invalid rosidl_type type. Needs to be one of {}'.format(
+            ROSIDL_FILTERS.keys()))
```
Comment (clalancette) [inline rqt_py_common/src/rqt_py_common/message_helpers.py]: This was wrong before, but I think it does give some information.  Thus I think this should be:

```suggestion
        raise ValueError('Invalid rosidl_type type {}. Needs to be one of {}'.format(
            rosidl_type, ROSIDL_FILTERS.keys()))
```

**Review by clalancette — CHANGES_REQUESTED** [review]
Two small changes, then this looks good.

**Comment by ahcorde** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/21392/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/15696/)
* Linux-rhel [Build Status](http://ci.ros2.org/job/ci_linux-rhel/899/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/22126/)

**Review by clalancette — APPROVED** [review]

**Pull request merged**